requirements.txt

```diff
@@ -1,17 +1,16 @@
-black==19.10b0
+black==21.7b0
 coverage==5.1
-flake8==3.7.9
+entrypoints==0.3
```

Comment:
I also wanted to comment to confirm that including the final run results (the `.xlsx` files) is intentional.